models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -437,7 +447,26 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
     if ("leafGrowth" %in% pft.names) {
       param[which(param[, 1] == "leafGrowth"), 2] <- pft.traits[which(pft.names == "leafGrowth")]
     }
-  }  ## end loop over PFTS
+
+    #update LeafOnday and LeafOffDay
+    if (update_phenology){
+     leaf_pheno_outdir <- settings$model$leaf_phenology$outdir  ## read from settings
+     if (!is.null(leaf_pheno_outdir)) {
+    ##read data
+       leafphdata <- utils::read.csv(paste0(leaf_pheno_outdir,"/leaf_phenology.csv"))
+       leafOnDay <- leafphdata$leafonday[leafphdata$year==obs_time & leafphdata$site_id==settings$run$site$id]
+       leafOffDay<- leafphdata$leafoffday[leafphdata$year==obs_time & leafphdata$site_id==settings$run$site$id]
```

Comment:
1. why does this have to be obs_time instead of start.date?
2. I would expect obs_time to be in a datetime format, and yet you're matching it against year? Seems like they're a missing extraction of year
3. how does SDA update the phenological status if the phenology is prescribed?
4. If you're adding a whole new class of inputs to the PEcAn system (prescribed phenology) there needs to be updates to the general documentation explaining any new tags, new sources, new file formats, etc. Indeed, there needs to be updates explaining how to run the sub-annual SDA more generally as well.